dl1_data_handler/utils.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+import ctapipe.image.cleaning as cleaning
+from ctapipe.instrument import CameraGeometry
+
+
+#################
+# Event Filters #
+#################
+
+def event_energy_filter(reader, file, e_min=-np.inf, e_max=np.inf):
```

Comment:
This isn't needed, since it can be efficiently implemented using the pytables cut condition `"({} < mc_energy) & (mc_energy < {})".format(e_min, e_max)`.